biz.aQute.bndlib/src/aQute/bnd/osgi/AnnotationHeaders.java

```diff
@@ -398,7 +400,7 @@ private void mergeAttributesAndDirectives(Annotation a) {
 		}
 
 		private void handleAttributeOrDirective(Annotation a) {
-			Object o = annotation.get(lastMethodSeen);
+			Object o = getOrDefault(lastMethodSeen);
```

Comment:
This violates the spec: https://osgi.org/specification/osgi.core/7.0.0/framework.api.html#org.osgi.annotation.bundle.Attribute.